R/create.R

```diff
@@ -39,21 +40,18 @@ create_package <- function(path,
 
   create_directory(path)
   old_project <- proj_set(path, force = TRUE)
+  on.exit(proj_set(old_project), add = TRUE)
 
   use_directory("R")
   use_directory("man")
   use_description(fields)
   use_namespace()
 
   if (rstudio) {
-    use_rstudio()
-  }
-
-  if (open) {
-    open_project(proj_get(), restore = old_project)
+    use_rstudio(open = open)
   }
 
-  invisible(TRUE)
+  invisible(proj_get())
```

Comment:
```suggestion
  invisible(path)
```

I think that's easier to reason about

Reply:
If we do this, we'll also need to re-process with `proj_path_prep()`, i.e. return `invisible(proj_path_prep(path))`. The reason for using what `proj_get()` returns is that it guarantees consistent path handling (realization, expansion, normalization, etc.). This, in turn, is necessary for other path comparison things to work, e.g. try to prevent more headaches like #479.